quic/s2n-quic-qns/etc/Dockerfile.build

```diff
@@ -41,7 +41,9 @@ RUN set -eux; \
   fi; \
   rm -rf target
 
-FROM martenseemann/quic-network-simulator-endpoint:latest
+# This quic-network-simulator-endpoint has an linux/arm64 image
+# to support running on ARM hardware (Apple M1, for example)
+FROM wesleyrosenblum/quic-network-simulator-endpoint:latest
```

Comment:
Would it be possible to upstream these changes?

Reply:
Yep, I can open a PR. The only downside I see is it causes the Docker build to take over 2 hours, but that is just a one time occurrence.

Reply:
https://github.com/marten-seemann/quic-network-simulator/pull/101

Reply:
The upstream change has been merged, and I've updated this PR